numpy/core/tests/test_umath.py

```diff
@@ -273,6 +273,12 @@ def test_floor_division_complex(self):
         y = np.floor_divide(x**2, x)
         assert_equal(y, [1.e+110, 0], err_msg=msg)
 
+    def test_floor_division_signed_zero(self):
+        # Check that the sign bit is correctly set when dividing positive and
+        # negative zero by one.
+        x = np.zeros(10)
+        assert_equal(np.signbit(x//1),0)
+        assert_equal(np.signbit((-x)//1),1)
```

Comment:
Can you add PEP8 conform spaces after the `,`? Spaces around `//` are optional.

Reply:
This has been updated in my latest push.